integration_tests/README.md

```diff
@@ -1,23 +1,90 @@
-###  Run the integration tests
+### Overview
+1. Prerequisites
+1. Configure credentials
+1. Setup Postgres (optional)
+1. Setup virtual environment
+1. Installation for development
+1. Run the integration tests
+1. Run tests
+1. Creating a new integration test
+
+### Prerequisites
+- python3
+- Docker
+
+### Configure credentials
+Edit the env file for your TARGET in `integration_tests/.env/[TARGET].env`.
+
+Load the environment variables:
+```shell
+set -a; source integration_tests/.env/[TARGET].env; set +a
+```
 
-To run the integration tests on your local machine, like they will get run in the CI (using CircleCI), you can do the following:
+or more specific:
+```shell
+set -a; source integration_tests/.env/postgres.env; set +a
```

Comment:
Is there any more preferable way to store and load environment variables containing secrets?

For non-secret project-specific environment variables, I like using [`direnv`](https://direnv.net/)